system/telnetd/Kconfig

```diff
@@ -40,6 +40,6 @@ config SYSTEM_TELNETD_SESSION_PRIORITY
 
 config SYSTEM_TELNETD_SESSION_STACKSIZE
 	int "Telnetd session task stack size"
-	default SYSTEM_NSH_STACKSIZE
+	default DEFAULT_TASK_STACKSIZE
```

Comment:
why not revert https://github.com/apache/nuttx-apps/pull/2706 or wait https://github.com/apache/nuttx-apps/pull/2716?